src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.props

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="'$(EnableDotnetAnalyzers)' == 'true'">
+    <!-- Enable the Microsoft.Dotnet.CodeAnalysis.dll analyzer --> 
+    <Analyzer Include="$(BuildToolsTaskDir)Microsoft.Dotnet.CodeAnalysis.dll"/>
```

Comment:
I would perfer if we know it is going to need to be a targets file (which I suspect it will) that we go ahead and do this as a targets file and plug it into the right build.common.targets location so that doesn't need to be updated later.